src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.input {
+  padding: 0 0 0 62px;
+  font-family: inherit;
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  height: 69px;
+  width: 100%;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  outline: none;
+  font-weight: 800;
+}
+
+.input::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+  position: relative;
+}
+
+.input-small {
+  height: 39px;
+  width: 100%;
+  border-radius: 4px;
+  padding: 0 0 0 33px;
+  font-size: 14px;
+}
+
+.input-small::placeholder {
+  font-size: 14px;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+  height: 70px;
+}
+
+.search-form-small {
+  position: relative;
+  margin-top: 20px;
+}
```

Comment:
add class `search-form` to small form as well
and in `search-form-small` just overwrite styles which are different from the original (height in this case)

do the same everywhere it's needed